src/backend.org

```diff
@@ -3305,6 +3352,52 @@ class TestNodalCentre(TestCase):
         self.assertEqual("IIIT-Hyd", centre.name)
 #+END_SRC
 
+
+***** setCentreStatus: NodalCentre, CentreStatus --> Void
+#+BEGIN_SRC python :tangle src/db.py :eval no 
+    @typecheck(centre_status=str)
+    def set_centre_status(self, centre_status):
+        self.centre_status = centre_status
+#+END_SRC
+      
+***** Test setName: NodalCentre, Name --> Void
```

Comment:
Python unit test case defined for the method setCentreStatus is wrong. This test case might fail while running. Please update it correctly.